draft-ietf-quic-transport.md

```diff
@@ -1133,23 +1133,35 @@ Servers MUST drop incoming packets under all other circumstances.
 
 ## Life of a QUIC Connection {#connection-lifecycle}
 
-TBD.
-
-<!-- Goes into how the next few sections are connected. Specifically, one goal
-is to combine the address validation section that shows up below with path
-validation that shows up later, and explain why these two mechanisms are
-required here.
-
-suggested structure:
-
- - establishment
-   - VN
-   - Retry
-   - Crypto
- - use (include migration)
- - shutdown
-
--->
+A QUIC connection is a stateful interaction between a client and server, the
+primary purpose of which is to support the exchange of data by an application
+protocol.  Streams ({{streams}}) is the primary mechanism by which an
```

Comment:
s/is/are?

Reply:
I really didn't know which here.  But I think that you are right.